File: spring/facemeet-backend-master/FaceMeet/src/main/java/com/levelup/FaceMeet/repository/user/FaceRepository.java
package com.levelup.FaceMeet.repository.user;

import com.levelup.FaceMeet.domain.face.Face;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface FaceRepository extends JpaRepository<Face, Long> {
}

File: android/FaceMeet/core/src/main/java/com/ssafy/facemeet/core/domain/repository/FaceRepository.kt
package com.ssafy.facemeet.core.domain.repository

import com.ssafy.facemeet.core.data.remote.dto.response.FaceAnalysisResponse
import com.ssafy.facemeet.core.data.remote.dto.response.FaceInfoResponse
import okhttp3.MultipartBody

interface FaceRepository {
    suspend fun analyzeFace(
        frontImage: MultipartBody.Part,
        sideImage: MultipartBody.Part
    ): Result<FaceAnalysisResponse>

    suspend fun getMyFaceInfo(): Result<FaceInfoResponse>
}
